src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,36 @@
-'use strict';
+"use strict";
 
 class Animal {
-  // write your code here
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+    Animal.alive.push(this);
+  }
 }
 
+Animal.alive = [];
+
 class Herbivore extends Animal {
-  // write your code here
+  constructor(name, health = 100) {
+    super(name, health);
+    this.hidden = false;
+  }
+
+  hide() {
+    this.hidden = true;
+  }
 }
 
 class Carnivore extends Animal {
-  // write your code here
+  bite(herbivore) {
```

Comment:
`bite(herbivore)` we do not know yet that animal is herbivore or carnivore. So a generic parameter name would be nicer.